app/views/shared/engines/_broker_menu.html.erb

```diff
@@ -14,7 +14,7 @@
         <% end %>
       </li>
       <li role="presentation" id="families-tab" <%= menu_tab_class(active_tab, "families-tab") %>>
-        <%= link_to(main_app.family_index_broker_agencies_profiles_path(id: @broker_agency_profile.id), "aria-expanded" => "true", :remote => true, "aria-controls" => "families", "role" => "tab" ) do  %>
+        <%= link_to(benefit_sponsors.family_index_profiles_broker_agencies_broker_agency_profiles_path(id: @broker_agency_profile.id), "aria-expanded" => "true", :remote => true, "aria-controls" => "families", "role" => "tab" ) do  %>
```

Comment:
Pointed to correct path, **new_model's** broker agency controller's family_index.